### PR TITLE
[change] Added geocoding check

### LINKED ISSUE #169: [change] Add geocoding check in deployment script
Geocoding tests are converted in deployment checks via [this PR](https://github.com/openwisp/django-loci/pull/97)

We should add this checks to ensure geocoding works as expected

Similar to https://github.com/openwisp/ansible-openwisp2/issues/310

### PR DESCRIPTION
**Pull request by Aryamanz29**
Closes #169
Similar to : https://github.com/openwisp/ansible-openwisp2/pull/379

### CODE REVIEW

File: build/common/init_command.sh [hunk 1]
```diff
@@ -7,6 +7,9 @@ init_conf
 
 # Start services
 if [ "$MODULE_NAME" = 'dashboard' ]; then
+    if [ "$OPENWISP_GEOCODING_CHECK" = 'True' ]; then
```
Comment (atb00ker) [inline build/common/init_command.sh:10]: Why are the permissions of the file changed from `0644` to `755`?

File: docs/ENV.md [hunk 2]
```diff
@@ -52,58 +52,58 @@ Additionally, you can search for the following:
```
Comment (atb00ker) [inline docs/ENV.md:52]: Wow, thanks!

File: build/common/init_command.sh (hunk 1 above)
Comment (Aryamanz29) [reply build/common/init_command.sh:10]: Maybe by mistake 😅, Could you please revert back.

Comment (Aryamanz29) [reply build/common/init_command.sh:10]: oops, by mistake :smiling_face_with_tear:

**Review by nemesisdesign — APPROVED** [review]

**Pull request merged**